tests/unit/type/string.tcl

```diff
@@ -394,6 +394,25 @@ start_server {tags {"string"}} {
         list $v1 $v2 [r get foo]
     } {{} OK 2}
 
+    test {Extended SET GET option} {
+        r del foo
+        r set foo bar
+        r set foo bar2 GET
+    } {bar}
+
+    test {Extended SET GET option with no previous value} {
+        r del foo
+        r set foo bar GET
+    } {}
+
+    test {Extended SET GET with NX option should result in syntax err} {
+      catch {r set foo bar NX GET} err1
+      catch {r set foo bar NX GET} err2
+      format $err1
+      format $err2
```

Comment:
what's the `format` lines are here for? (maybe something i don't know about tcl?

Reply:
honestly, I don't know why it should be here, but all tests catching errors are using this syntax: https://github.com/redis/redis/blob/unstable/tests/unit/type/string.tcl#L139
https://github.com/redis/redis/blob/unstable/tests/unit/type/incr.tcl#L30
and other places...

Reply:
i think they're using it to set a value that can be read by the test at exit.
like doing
```tcl
test {bla} {
    catch {something} err
    set _ $err
} {*pattern*}
```
i.e. `format` returns a value, and in this case we're not even storing it.
i think they're useless in your case (unless there's something i don't yet know about tcl)

Reply:
Understood! I remove the format and all the tests passed.